pyface/wx/image_control.py

```diff
@@ -0,0 +1,59 @@
+# (C) Copyright 2004-2023 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+""" Defines a wxPython ImageControl widget that is used by various Widgets.
+"""
+
+
+import wx
+
+
+class ImageControl(wx.Window):
```

Comment:
If I wanted to see this code in action, what's the easiest way for me to exercise this code manually? Are there examples that make use of this?

Reply:
No examples currently, but this was copied and simplified from the wx TriatsUI ImageEditor's ImageControl: https://github.com/enthought/traitsui/blob/main/traitsui/wx/image_control.py

Reply:
You could exercise it with something like:
```
import wx
from pyface.api import ImageResource
from pyface.wx.image_control import ImageControl

window = wx.Frame()
# hack to get a wx bitmap - could also get it through wx api and an image file
bitmap = ImageResource('splash').create_bitmap()
control = ImageControl(window, bitmap)
window.Show()
```
I think (my wx is rusty)